README.rst

```diff
@@ -56,6 +56,10 @@ Superuser passwords are:
 - **AMC:** ``amc`` with email ``amc@example.com`` and password ``amc``.
 - **Studio and LMS superuser:** ``edx`` with email ``edx@example.com`` and password ``edx``.
 
+
+    You'll also have a normal user in ``edx`` for ``amc`` with email ``amc@example.com`` and password ``amc``
```

Comment:
@ahmedaljazzar there's already an `amc:amc` user that is superuser. I think this change is not needed.

Reply:
That's in AMC, what I'm trying to do here, is to add `amc` user to ex-platform as well

Reply:
But we already have an `edx:edx` user, why adding another similarly powerful user?